package.json

```diff
@@ -11,15 +11,15 @@
     "bellhop-iframe": "^2.1.2",
     "lodash-es": "^4.17.10",
     "quill": "^1.3.6",
+    "springroll": "git+https://github.com/SpringRoll/SpringRoll.git#v2",
```

Comment:
Let's point at a different branch. I want to blow away that v2 branch.

Reply:
Updated to just point at the default branch, assuming that's ok

Reply:
Update to point at 2.2.0